### ISSUE TITLE
terminology issues

### ISSUE THREAD

**Issue by davidwhogg** [mkness/TheCannon#38]
Are we *completely consistent* at this point on terminology?  Are they "objects" or "stars" or "spectra"?  Are they "training" or "reference"?  Are they "survey" objects?

**Comment by mkness** [comment]
- We are not completely consistent, I have made some modifications but propose we do not have/want to be using the same terms in all sections e.g. general versus specific APOGEE case. 
- I will leave this issue open as we should revisit this in an overall pass at the end:

- stars, spectra, objects: in the general discussion I changed 'star/spectra' to 'objects' and in other parts of the paper I think spectra and stars is fine - using one or the other emphasises the overall object v. use of the information.
-training/reference objects: I am trying to be consistent within sections e.g. general v. specifically APOGEE - but allowing both terms is ok. 
- survey objects is used to describe the test objects.
- one issue that numerous people at the AAS came up to me about was the use of 'labels' rather than e.g. stellar parameters; they found this confusing to use this sort of term (I did define it but there was still a lot of questioning about why this is necessary). We all agree that this is the appropriate terminology  - this forces people to think about the problem generally. We do make this definition clear in the paper - perhaps we could say it in another way elsewhere as well although this is repetition.
